src/main/resources/jsonSchema/clientSettings.schema.json

```diff
@@ -11,6 +11,12 @@
       "description": "Whether or not the language grammar is case-sensitive.",
       "default": false
     },
+    "statementTerminatorCharacters": {
```

Comment:
Client config JSON schema updates for language grammar-level statement terminator characters and editor behavior features.